include/oneapi/dpl/optional

```diff
@@ -24,10 +24,10 @@ namespace oneapi
 namespace dpl
 {
 #if __cplusplus > 201402L
-using ::std::make_optional;
-using ::std::nullopt;
-using ::std::nullopt_t;
-using ::std::optional;
+using ::std::make_optional;     // KSATODO: leave as is: https://en.cppreference.com/w/cpp/utility/optional/make_optional - since C++17
+using ::std::nullopt;           // KSATODO: leave as is: https://en.cppreference.com/w/cpp/utility/optional/nullopt - since C++17
+using ::std::nullopt_t;         // KSATODO: leave as is: https://en.cppreference.com/w/cpp/types/nullptr_t - since C++17
+using ::std::optional;          // KSATODO: leave as is: https://en.cppreference.com/w/cpp/utility/optional - since C++17
 #endif // __cplusplus > 201402L
```

Comment:
Why keeping the "greater than C++14" condition check?

Reply:
Fixed: removed support of C++14 too